x-pack/plugins/timelines/server/search_strategy/timeline/factory/events/details/query.events_details.dsl.ts

```diff
@@ -40,6 +48,8 @@ export const buildTimelineDetailsQuery = (
       docvalue_fields: docValueFields,
       query,
       fields: [{ field: '*', include_unmapped: true }],
+      // Remove and instead pass index_pattern.id once issue resolved: https://github.com/elastic/kibana/issues/111762
+      runtime_mappings: runtimeMappings,
```

Comment:
we'll be able to remove `runtimeMappings` from everywhere once this `priority: low`/`loe: hours` issue is completed: https://github.com/elastic/kibana/issues/111762